teckers/teckers/GoogleService-Info.plist

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file should be uploaded to repo, please remove it from PR

Reply:
I removed this